.circleci/config.yml

```diff
@@ -209,6 +212,8 @@ jobs:
           command: |
             cd examples/prelude
             cmd.exe /c mklink /D prelude C:\Users\circleci\project\prelude
+            # Skip Conan tests on Windows. TODO[AH] Implement Windows support.
+            Remove-Item -Recurse -Force cpp/conan
```

Comment:
We usually do this with host_info if's in the prelude - see where we do it in other places.

Reply:
The difficulty here is that some of those targets are generated, namely those in `examples/prelude/cpp/conan/import/BUILD`. See also the internal discussion we had on this from 21/12/2022. Removing the folder seemed like the best option at the time. But, I'm happy to do something else if there's a better option.